config/mock-api-data.js

```diff
@@ -11,6 +11,26 @@ function * range (max)
     return max;
 }
 
+function setTagName () {
+    
+    const tagsArray = [
```

Comment:
We should consider storing the tags in db as ```id, name``` pair. @Bartoshko @ksiadz @4fuss Thoughts?

Reply:
Not outdated

Reply:
Yes, tags are only names with many2many relation to store. If you want pass many tags just repeat the get parameter.